### PR TITLE
DEM Bugfix Virtual Sea Surface

### PR DESCRIPTION
**Pull request by salvalatorre**
The Virtual Sea Surface info was missing in the default DEM parameters

### CODE REVIEW

**Comment by salvalatorre** [comment]
Added a subsection in the json for the virtual sea surface settings. Made the necessary adjustments in the problem type

**Review by maceligueta — CHANGES_REQUESTED** [review]
Great! You are the first at indentating the json in DEM! Looks nice. Just a question, is there an option to activate/deactivate this virtual surface? If so, it can be put inside this block.

File: applications/DEM_application/python_scripts/DEM_procedures.py [hunk 1]
```diff
@@ -1464,15 +1466,16 @@ def __init__(self, DEM_parameters, post_path):
...
+        self.SeaSurfaceX1 = self.DEM_parameters["virtual_sea_surface_settings"]["PostVirtualSeaSurfaceX1"].GetDouble()
+        if self.SeaSurfaceX1:
+            self.SeaSurfaceY1 = self.DEM_parameters["virtual_sea_surface_settings"]["PostVirtualSeaSurfaceY1"].GetDouble()
+            self.SeaSurfaceX2 = self.DEM_parameters["virtual_sea_surface_settings"]["PostVirtualSeaSurfaceX2"].GetDouble()
+            self.SeaSurfaceY2 = self.DEM_parameters["virtual_sea_surface_settings"]["PostVirtualSeaSurfaceY2"].GetDouble()
+            self.SeaSurfaceX3 = self.DEM_parameters["virtual_sea_surface_settings"]["PostVirtualSeaSurfaceX3"].GetDouble()
+            self.SeaSurfaceY3 = self.DEM_parameters["virtual_sea_surface_settings"]["PostVirtualSeaSurfaceY3"].GetDouble()
+            self.SeaSurfaceX4 = self.DEM_parameters["virtual_sea_surface_settings"]["PostVirtualSeaSurfaceX4"].GetDouble()
+            self.SeaSurfaceY4 = self.DEM_parameters["virtual_sea_surface_settings"]["PostVirtualSeaSurfaceY4"].GetDouble()
+            self.PrintVirtualSeaSurface = True
```
Comment (maceligueta) [inline applications/DEM_application/python_scripts/DEM_procedures.py]: _self.DEM_parameters["virtual_sea_surface_settings"]_ is repeated many times, you can save it in a local variable like:
`sea_settings = self.DEM_parameters["virtual_sea_surface_settings"]`
then you can do:
`self.SeaSurfaceY1 = sea_settings ["PostVirtualSeaSurfaceY1"].GetDouble()`
But I also wonder if it is really necessary to save self.SeaSurfaceY1 if we can access sea_settings ["PostVirtualSeaSurfaceY1"].GetDouble() any time we want. Your call on the last comment.

**Comment by salvalatorre** [comment]
Bug corrected, thanks!

**Comment by maceligueta** [comment]
The abreviation sea_settings = self.DEM_parameters["virtual_sea_surface_settings"] is still pending. I'll approve after that. Please make sure the examples run, I don't have any at hand right now (and any case without the sea as well!).

**Comment by salvalatorre** [comment]
Wait, benchmarks don't work, only GiD benchmarks...

**Review by maceligueta — APPROVED** [review]

**Pull request merged**